.github/workflows/test-java.yaml

```diff
@@ -13,14 +13,71 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
+  build-prql-lib:
```

Comment:
It might be worth splitting this out into a separate workflow that both `test-java` and `release` call and the pick up the artifact, to reduce duplication. 

But it's totally fine to add a `TODO` in the code and we can come back to it in a future PR

Reply:
Thanks. I realized that when copying it from test-java to release 😓  Will try to reuse that in a separate workflow.